lib/matplotlib/backends/backend_pdf.py

```diff
@@ -452,7 +481,7 @@ def __init__(self, filename, metadata=None):
         self.pagesObject = self.reserveObject('pages')
         self.pageList = []
         self.fontObject = self.reserveObject('fonts')
-        self.alphaStateObject = self.reserveObject('extended graphics states')
+        self.extGStateObject = self.reserveObject('extended graphics states')
```

Comment:
either keep the old name, or make the new name private, or at least document the change.